codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameCodec.java

```diff
@@ -96,6 +97,15 @@ protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) t
     }
 
     @Override
+    public void channelReadComplete(ChannelHandlerContext ctx) throws Exception {
+        if (read) {
+            // something was read we should notify the read of handlers in the pipeline.
```

Comment:
Please focus on punctuation. :-)

Something was read. We should notify the next handlers in the pipeline.